mdx-layout/src/NavBar.scss

```diff
@@ -1,6 +1,9 @@
 nav {
   h3 {
+    width: 100%;
     display: flex;
+    justify-content: space-between;
```

Comment:
Initially there was a span rule with a flex-grow:1, we can I think either remove the span rule because of this, or we can change to target :firstchild instead of span with flex grow